src/main/java/com/example/KitchenProject/Ingredient.java

```diff
@@ -0,0 +1,14 @@
+package com.example.KitchenProject;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@AllArgsConstructor
```

Comment:
The annotation from `lombok` I was talking about earlier is `@Data`. You can read more about it here: https://projectlombok.org/features/Data